js/about/bookmarks.js

```diff
@@ -106,7 +109,9 @@ class BookmarkFolderList extends ImmutableComponent {
       }
       {
         this.props.bookmarkFolders.map((bookmarkFolder) =>
-          <BookmarkFolderItem bookmarkFolder={bookmarkFolder}
+          this.props.isRoot && bookmarkFolder.get('parentFolderId') === -1
+          ? null
+          : <BookmarkFolderItem bookmarkFolder={bookmarkFolder}
             allBookmarkFolders={this.props.allBookmarkFolders}
```

Comment:
This check is so that folders that are children of "Other Bookmarks" don't show under "Bookmarks Toolbar".  Since the above check (line 50) was removed, it will now bind ALL bookmarks to this control.